src/features/programSample/programSample.ts

```diff
@@ -25,12 +25,37 @@ export type SampleProgress = {
     progressJson: Progress;
 };
 
+export type SupportedDeviceVersion = 'nRF9160' | 'nRF9161';
+
+export const getNrfDeviceVersion = (
+    device?: Device
+): SupportedDeviceVersion => {
+    let deviceVersion: 'nRF9160' | 'nRF9161' | undefined;
+    if (is9161DK(device)) {
+        deviceVersion = 'nRF9161';
+    } else if (is9160DK(device) || isThingy91(device)) {
+        deviceVersion = 'nRF9160';
+    }
+    if (deviceVersion) {
+        return deviceVersion;
+    }
```

Comment:
Not sure here but would it maybe be better to use deviceInfo(device).name here? 
Example of the format: https://github.com/NordicSemiconductor/pc-nrfconnect-shared/blob/main/src/Device/deviceInfo/deviceInfo.ts#L197

I see that the usecase is different but I figured we should try to stick with one convention cross-app as much as possible.

Reply:
Hmm, this is a valid point. We are guaranteed to always get this info as long is it's that device, right?

Reply:
I would think so, yeah.